projects/illm/default_transforms.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torchvision.transforms import (
+    CenterCrop,
+    Compose,
+    RandomChoice,
+    RandomCrop,
+    RandomHorizontalFlip,
+    RandomResizedCrop,
+    Resize,
+    ToTensor,
+)
+
+
+def default_train_transform(image_size: int) -> Compose:
+    return Compose(
+        [
+            RandomChoice(
+                [
+                    RandomCrop(
```

Comment:
I tend to avoid nesting to much and having that amount of indent for transforms, mostly by assigning to intermediate variables - not sure what's best thought

Reply:
I removed a little bit of indentation.